app/views/content/train-to-be-a-teacher/who-do-you-want-to-teach.md

```diff
@@ -3,114 +3,85 @@ title: Who do you want to teach
 
-Once you’re a qualified teacher you’ll need to complete the National Award in Special Educational Needs Coordination (NASENCo) when you take up your SENCO post.
+If you’d prefer to specialise in a subject but still enjoy teaching a variety of things, being a secondary school teacher might suit you.
 
-## Early years teacher training
+### What you need to teach in primary and secondary schools
 
-The early years are critical in children’s development. Your input will have a lasting and positive effect on their development and wellbeing. You’ll use your knowledge and skills to help young children enjoy high standards of teaching and open their minds to new ideas every day.
+To teach in the majority of primary and secondary schools, you will need to achieve qualified teacher status (QTS) and there are a number of routes to this. Check [what training you need to get QTS](/train-to-be-a-teacher).
 
-You’ll need to do early years initial teacher training (EYITT) to achieve early years teacher status (EYTS).
+Once you have QTS, you're qualified to teach any of these age ranges. If you want to change age groups you'll need to be able to show you have relevant classroom experience, for example being able to cover the broader curriculum if you want to switch from secondary to primary.
 
-EYTS is different from QTS, which is the equivalent for teaching children aged 3 to 18.
+### How to decide which age group to teach
 
-Your training will enable you to meet the [early years teachers’ standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+The best way to work out which age group you would like to teach is to get real life experience in a classroom. 
 
-To start early years teacher training, you’ll need these qualifications (or to demonstrate an equivalent standard):
+$get-school-experience$
 
-- maths grade 4 (C)
-- English grade 4 (C)
-- science grade 4 (C)
+If you’re an undergraduate and interested in teaching chemistry, computing, languages, maths or physics then you could do a [paid teaching internship](/teaching-internship-providers) to find out if teaching is right for you.
 
-If you do postgraduate early years teacher training, you’ll need a 2:2 bachelor’s degree with honours or higher.
+### Pupils with special educational needs
+Most disabled pupils and pupils with special educational needs learn in mainstream schools. Wherever you train and work – whether it’s mainstream schools, special schools, or both – you’ll work with pupils with a range of needs.
 
-[Contact accredited early years training providers to apply for a place](https://www.gov.uk/government/publications/early-years-initial-teacher-training-itt-providers-and-school-direct-early-years-lead-organisations/early-years-initial-teacher-training-itt-providers-and-school-direct-lead-organisations).
+You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train and teach.
```

Comment:
```suggestion
You can choose to apply for a teacher training course that specialises in SEND or you can gain the skills and experience as you train to teach.
```
(not quite sure whether this is the right change!)

Reply:
SEND hasn't been defined at this point so we shouldn't use an acronym?

could change the h3 to add in disabilities (should do this anyway) and then the SEND acronym?